src/kontrol/__main__.py

```diff
@@ -69,12 +69,10 @@ def _ignore_arg(args: dict[str, Any], arg: str, cli_option: str) -> None:
 def _load_foundry(foundry_root: Path, bug_report: BugReport | None = None, use_hex_encoding: bool = False) -> Foundry:
     try:
         foundry = Foundry(foundry_root=foundry_root, bug_report=bug_report, use_hex_encoding=use_hex_encoding)
-    except FileNotFoundError:
-        print(
+    except FileNotFoundError as err:
+        raise RuntimeError(
             f'File foundry.toml not found in: {str(foundry_root)!r}. Are you running kontrol in a Foundry project?',
-            file=sys.stderr,
-        )
-        sys.exit(1)
```

Comment:
This was introduced in #204, which fixed #93. In this PR, I'd just change the exit code.

Reply:
Right, thank you! I've reverted this change and instead changed the exit code to `127` since it's a "file not found" issue: https://github.com/runtimeverification/kontrol/pull/510/commits/9d8e7400726f2a43fe2bfb70d24ce28b6cfd27a1.